### ISSUE TITLE
[Essentials] /vanish

### ISSUE THREAD

**Issue by enterprise12nx** [keepcalm/BukkitForge#450]
My mods are using /vanish from essentials, they say people can still see them? i will continue to test!

**Comment by gravypod** [comment]
this might be a problem still, We will see what we can do.

**Comment by enterprise12nx** [comment]
ok thanks!
Hope all my bug reports help.

**Comment by eagl3s1ght** [comment]
I'll add what I wrote in IRC.

I've had minor success with vanish. It hides the user (but holding an item shows the item, and some hat items show), but trying to cancel vanish disconnects the user. Also, while vanish hides the user, there is still the effect of invisibility (some kind of bubble effect) which anyone can recognize to see where the vanished player is.

**Comment by gravypod** [comment]
If you are using Essentials /vanish it grants the vanish potion.

**Comment by EDSman11** [comment]
I seem to be having the same problem except for me the /vanish command does nothing and when I do /v it tells me "You have been vanished!" then I type /vanish to unvanish but instead it says "You have been vanished!" again as if they were two entirely different commands...

**Comment by gravypod** [comment]
@EDSman11 They are. Essentials has /vanish IIRC, something else might have /v

**Comment by EDSman11** [comment]
No, they are both part of essentials, /v seems to be an alias for /vanish...

**Comment by eagl3s1ght** [comment]
> I seem to be having the same problem except for me the /vanish command does nothing and when I do /v it tells me "You have been vanished!" then I type /vanish to unvanish but instead it says "You have been vanished!" again as if they were two entirely different commands...

@EDSman11, that's not how it works. [Essentials Command Reference](http://ess.khhq.net/wiki/Command_Reference) says you must use /v off to disable the effect...

**Comment by EDSman11** [comment]
Ok, good to know...

**Comment by eagl3s1ght** [comment]
Seems **partially** fixed with build 223. Now gives no Potion effect when using /vanish. Held blocks and armor is still shown though.

**Edit:** I'm not sure anymore. A user noticed I was stalking him by saying "a invisible player is here". I am seeing no potion effect on myself when using though.

**Comment by EDSman11** [comment]
Do I need to have a permissions plugin running for non-ops to not be able
to see vanished players?

**Comment by eagl3s1ght** [comment]
It could be so, yes. Because of the permission node **essentials.vanish.see**.

> Do I need to have a permissions plugin running for non-ops to not be able
> to see vanished players?
